ods_ci/tests/Tests/1000__model_serving/1010__model_serving_triton_on_modelmesh/1010__model_serving_triton_on_modelmesh.robot

```diff
@@ -0,0 +1,100 @@
+*** Settings ***
+Documentation     Suite of test cases for Triton in Modelmesh
+Library           OperatingSystem
+Library           ../../../../libs/Helpers.py
+Resource          ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource
+Resource          ../../../Resources/Page/ODH/Monitoring/Monitoring.resource
+Resource          ../../../Resources/OCP.resource
+Resource          ../../../Resources/CLI/ModelServing/modelmesh.resource
+Resource          ../../../Resources/Common.robot
+Suite Setup       Triton On Kserve Suite Setup
+Suite Teardown    Triton On Kserve Suite Teardown
+Test Tags         Kserve
+
+
+*** Variables ***
+${INFERENCE_REST_INPUT_ONNX_FILE}=    @tests/Resources/Files/triton/kserve-triton-onnx-rest-input.json
```

Comment:
you could use `${RESOURCES_DIRPATH}` inside this path

Reply:
This variable ${RESOURCES_DIRPATH} doesn't require @. If I add it, it doesn't work. Since it doesn’t need @ for other cases, I left it out

Reply:
```code
${INFERENCE_REST_INPUT_ONNX_FILE}=     @${RESOURCES_DIRPATH}/kserve-triton-onnx-rest-input.json
```